README.md

```diff
@@ -2,4 +2,119 @@
+| `auth.ClientKeyFilePath`    | Path for key file                                                                                                                                                                               | Required for X509 type.                   | /tmp/key.cert                                  |
+
+### Snapshot
+By default when the connector starts for the first time, snapshot mode is enabled, which means that existing data will
+be read. To skip reading existing, change config parameter `snapshot` to `false`.
+
+First time when the snapshot iterator starts work,
+it is get max value from `orderingColumn` and saves this value to position.
+The snapshot iterator reads all rows, where `orderingColumn` values less or equal maxValue, from the table in batches.
+
+
+Values in the ordering column must be unique and suitable for sorting, otherwise, the snapshot won't work correctly.
+Iterators saves last processed value from `orderingColumn` column to position to field `SnapshotLastProcessedVal`.
+If snapshot was interrupted on next start connector will parse last recorded position
+to find next snapshot rows.
+
+
+When all records are returned, the connector switches to the CDC iterator.
+
+### Change Data Capture (CDC)
+
+This connector implements CDC features for DB2 by adding a tracking table and triggers to populate it. The tracking
+table has the same name as a target table with the prefix `CONDUIT_` and suffix from time when pipeline started on
+format "hhmmss". For example for table `PRODUCTS` tracking will be looks like `CONDUIT_PRODUCTS_213315`. 
+The tracking table has all the same columns as the target table plus two additional columns:
+
+| name                            | description                                      |
+|---------------------------------|--------------------------------------------------|
+| `CONDUIT_TRACKING_ID`           | Autoincrement index for the position.            |
+| `CONDUIT_OPERATION_TYPE`        | Operation type: `INSERT`, `UPDATE`, or `DELETE`. |
+
+The connector saves  information about update, delete, insert `table` operations inside tracking table.
+For example if user inserts new row into `table` connector will save all new columns values inside tracking table  
+with `CONDUIT_OPERATION_TYPE` = `INSERT`
+
+Triggers have name pattern `CD_{{TABLENAME}}_{{OPERATION_TYPE}}_{{SUFFIXNAME}}`. For example:
+`CD_PRODUCTS_INSERT_213315`
+
+
+Queries to retrieve change data from a tracking table are very similar to queries in a Snapshot iterator, but with
+`CONDUIT_TRACKING_ID` ordering column.
+
+CDC iterator periodically clears rows which were successfully applied from tracking table.
+It collects `CONDUIT_TRACKING_ID` inside the `Ack` method into a batch and clears the tracking table every 5 seconds.
```

Comment:
IMHO we can summarize this into one sentence and say that the connector cleans up the tracking table every 5 seconds (is this configurable btw?). We don't need to go into details how that happens.

Reply:
Changed.
This is not configurable value  like in DB2  source connector.